### DISCUSSION TITLE
Stuck at "Initializing auth module: /opt/AdGuardHome/data/sessions.db"

### DISCUSSION THREAD

**Discussion by PietroMarzio** [AdguardTeam/AdGuardHome#6441] [Q&A]
Hello guys,
I am trying to install AGH on an MX Linux. I followed the wiki until the moment of installation process.
Running ./AdGuardHome , the process is stuck and it does not proceed.

`root@mx:/opt/AdGuardHome# ./AdGuardHome 
2023/11/19 16:56:54.875828 [info] AdGuard Home, version v0.107.41
2023/11/19 16:56:54.876724 [info] tls: using default ciphers
2023/11/19 16:56:54.878156 [info] safesearch default: disabled
2023/11/19 16:56:54.883863 [info] Initializing auth module: /opt/AdGuardHome/data/sessions.db

`

Researches on old topics done, I do not understand what is missing.

Thanks for your help!

**Comment by PietroMarzio** [comment]
No one had the same issue?

**Reply by ndelb** [reply]
I seem to have the same issue as you.

**Reply by phrequencyviii** [reply]
Same problem here on MacOS Sonoma

**Reply by chris-lansman** [reply]
On  Operating System: Ubuntu 20.04.6 LTS
       Kernel: Linux 5.15.0-91-generic

Having the same issue when I run sudo ./AdGuardHome

**Reply by 99999kcal** [reply]
me too in sonoma

**Reply by 7shit** [reply]
> No one had the same issue?

i hv same problem, just kill agduard in task manager or in cmd stop the adguard services and run again...

**Comment by Neurothiker** [comment]
same here:
openwrt
root@GL-MT6000:~# logread -e "adguard"
Sun Dec  1 08:17:43 2024 daemon.err AdGuardHome[7110]: 2024/12/01 07:17:43.020667 [info] Initializing auth module: /tmp/adguardhome/data/sessions.db

**Comment by tuantemtinh** [comment]
same here. with debian 12
